src/models/__init__.py

```diff
@@ -3,3 +3,4 @@
 db = SQLAlchemy()
 
 from .customer import Customer
+from .payment_methods import PaymentMethods
```

Comment:
the class name is supposed to be in a singular form.

from .payment_methods import PaymentMethod